docs/features/grok-oauth/plan.md

```
@@ -0,0 +1,56 @@
+
+| Item | Value |
+| --- | --- |
+| Issuer | `https://auth.x.ai` |
+| Discovery | `/.well-known/openid-configuration` |
+| Client ID | public Grok CLI client `b1a00492-073a-47ea-816f-4c329264a828` |
+| Scope | `openid profile email offline_access grok-cli:access api:access` |
+| API base | `https://api.x.ai/v1` |
+
+## Data Flow
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)